cl/search/tests/tests_es_oral_arguments.py

```diff
@@ -2189,7 +2189,8 @@ def test_stemming_disable_search(self) -> None:
         self.assertEqual(actual, expected)
         self.assertIn("Freedom of", r.content.decode())
         self.assertIn("<mark>Inform</mark>", r.content.decode())
-        self.assertEqual(r.content.decode().count("<mark>Inform</mark>"), 1)
+        print(" r.content.decode()", r.content.decode())
```

Comment:
This seems to have been added accidentally

Reply:
Thanks, removed!